middlewares/verifyAdmin.js

```diff
@@ -0,0 +1,8 @@
+const verifyAdmin = (req, res, next) => {
+    if (req.user.roleId !== 1) {
```

Comment:
Tener roles almacenados en constantes nombradas, no utilizar valores sueltos, por ejemplo:
```
const ADMIN_ROLE = 1;
const STANDARD_ROLE = 2;
```
y despues en codigo:

```if (req.user.roleId !== ADMIN_ROLE) ...```

Reply:
I put the admin_role info in the .env file